.github/workflows/test.yml

```diff
@@ -102,6 +102,9 @@ jobs:
           sudo curl -L https://github.com/docker/compose/releases/download/${{ matrix.compose_version }}/docker-compose-`uname -s`-`uname -m` -o "${{ matrix.compose_path }}/docker-compose"
           sudo chmod +x "${{ matrix.compose_path }}/docker-compose"
 
+      - name: Ensure runner is owner of /var/lib/docker to set permissions of docker volumes
```

Comment:
Unfortunately, this is needed since the github runner does not have access to `/var/lib/docker/volumes` and I'd like to avoid invoking sudo in the install script

Reply:
this is specific since we're performing a chmod now in the install script